x-pack/plugins/monitoring/public/components/summary_status/summary_status.js

```diff
@@ -52,7 +68,8 @@ const StatusIndicator = ({ status, isOnline, IconComponent }) => {
   );
 };
 
-export function SummaryStatus({ metrics, status, isOnline, IconComponent = DefaultIconComponent, ...props }) {
+// eslint-disable-next-line no-unused-vars
+export function SummaryStatus({ metrics, status, isOnline, IconComponent = DefaultIconComponent, intl, ...props }) {
```

Comment:
we add '**intl**' here because in the test for **SummaryStatus** we use '**renderWithIntl**'.
https://github.com/elastic/kibana/pull/24973/files#diff-9e2803564a1fa0f70e630c1d9f61814fR29
'**renderWithIntl**' add '**intl**' in the props. 
If we don't add '**intl**' as argument, '**intl**' drops to ...props here https://github.com/elastic/kibana/pull/24973/files#diff-e87682e532627ab40213f0d0af2dbb95R75
and we are getting unexpected result in snapshot.